packages/react-core/src/index.ts

```diff
@@ -20,30 +20,7 @@ export {
   UseAuthenticatorRoute,
 } from './Authenticator';
 
-export {
-  BannerMessageCommonProps,
-  BannerMessageComponent,
-  CarouselMessageCommonProps,
-  CarouselMessageComponent,
-  FullScreenMessageCommonProps,
-  FullScreenMessageComponent,
-  handleMessageAction,
-  HandleMessageLinkAction,
-  InAppMessagingProvider,
-  MessageButtonProps,
-  MessageCommonProps,
-  MessageComponentBaseProps,
-  MessageContentProps,
-  MessageImage,
-  MessageLayout,
-  MessagePayloadStyle,
-  MessageTextAlign,
-  ModalMessageCommonProps,
-  ModalMessageComponent,
-  OnMessageAction,
-  useInAppMessaging,
-  useMessage,
-} from './InAppMessaging';
+export { RenderNothing } from './components';
```

Comment:
Not blocking but does this even need to be exported? I can probably go either way but seems like the component doesn't really do much.